Tries1.java

```diff
@@ -0,0 +1,43 @@
+//Time complexity - O(M) - M is the key length
+//Space complexity - O(M)
```

Comment:
Space, would be O(M* L), where L is is the average length of the words in the Trie.